packages/zoltan2/src/algorithms/partition/Zoltan2_AlgMultiJagged.hpp

```diff
@@ -2076,7 +2068,7 @@ void AlgMJ<mj_scalar_t, mj_lno_t, mj_gno_t, mj_part_t, mj_node_t>::
     this->part_xadj = this->new_part_xadj;
     this->host_part_xadj = Kokkos::create_mirror_view(part_xadj);
     Kokkos::deep_copy(host_part_xadj, part_xadj); // keep in sync
-    this->new_part_xadj = Kokkos::View<mj_lno_t*, device_t>("empty");
+    this->new_part_xadj = Kokkos::View<mj_lno_t*, device_t>("empty", 0);
```

Comment:
You don't need to allocate empty Views.  Just let them be default constructed.

Reply:
Deprecated off won't allow the constructor with label and no dimensions: view_t("empty"), but will allow the default constructor: view_t() or with size: view_t("empty",0). The latter would provide the label for a debug out of bounds error which seems potentially useful. Wondering now if there might be a utility to allowing the constructor with label only.

@kddevin if we change to default constructors I could do it on a subsequent pass as I expect there will be further updates. Whether we keep the label does not seem critical.